cfgov/unprocessed/css/enhancements/tables.less

```diff
@@ -2,6 +2,24 @@
   name: Tables
   family: cf-tables
   notes:
+    - A fix that makes stacking cells 100% width when stacked
+  tags:
+    - cfgov-cf-tables
+*/
+
+.respond-to-max(@bp-xs-max, {
+    .o-table__stack-on-small {
```

Comment:
I think you can reuse https://github.com/cfpb/cfgov-refresh/blob/27cbbaa92e4c9d19799b80f06e6b7013217129de/cfgov/unprocessed/css/enhancements/tables.less#L144.